warehouse/keyword-common/src/main/java/datawave/util/keyword/TagCloudPartition.java

```diff
@@ -0,0 +1,148 @@
+package datawave.util.keyword;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInput;
+import java.io.DataInputStream;
+import java.io.DataOutput;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import org.apache.hadoop.io.Writable;
+
+/**
+ * Used to partition TagCloudInput
+ */
+public class TagCloudPartition implements Writable {
+    public enum SCORE_TYPE {
```

Comment:
Consider `ScoreType` instead of `SCORE_TYPE`. I see that we use either convention in datawave, but UpperCampleCase seems more appropriate for enums because they behave like classes generally.

```suggestion
    public enum ScoreType {
```